config/seata-config-nacos/src/main/java/io/seata/config/nacos/NacosConfiguration.java

```diff
@@ -94,19 +101,26 @@ public String getConfig(String dataId, String defaultValue, long timeoutMills) {
         if ((value = getConfigFromSysPro(dataId)) != null) {
             return value;
         }
-        try {
-            value = configService.getConfig(dataId, getNacosGroup(), timeoutMills);
-        } catch (NacosException exx) {
-            LOGGER.error(exx.getErrMsg());
+
+        value = seataConfig.getProperty(dataId);
+
+        if (null == value) {
```

Comment:
It is recommended to put this value in seataConfig, and get it from seataConfig next time.